src/string_view.h

```diff
@@ -24,19 +24,22 @@
 
 using StringView = lcf::StringView;
 
+using lcf::ToString;
+using lcf::ToStringView;
 using lcf::StartsWith;
 using lcf::EndsWith;
 using lcf::Substr;
-using lcf::ToString;
-
-namespace lcf {
 
 // FIXME: liblcf doesn't depend on fmt, so we need to add this here to enable fmtlib support for our StringView.
-template <typename C>
-inline fmt::basic_string_view<C> to_string_view(BasicStringView<C> s) {
-	return fmt::basic_string_view<C>(s.data(), s.length());
+namespace nonstd { namespace sv_lite {
+template <typename C, typename T>
+inline fmt::basic_string_view<C> to_string_view(basic_string_view<C,T> s) {
+    return fmt::basic_string_view<C>(s.data(), s.size());
 }
+} }
+
```

Comment:
Highlighting these ugly hacks to get fmtlib working without needing to include `<fmt/format.h>` everywhere

Reply:
any hack that doesn't increase buildtime by 100% because of libfmt is good.